azure-pipelines.yml

```diff
@@ -241,6 +241,11 @@ stages:
             TestResultsArtifactName: 'DotNetTestResultsTestFramework'
             ProjectFilePath: 'tests\SonarAnalyzer.TestFramework.Test\SonarAnalyzer.TestFramework.Test.csproj'
             FrameworkMoniker: 'net7.0-windows'
+          ITs.JsonParser:
```

Comment:
You also need to restore it later around line 386 and 410

Reply:
It would actually be handy to call the `Execute` from test, without really asserting anything. Just to see the coverage working.

Reply:
I wanted to do that, but top level statement files do not allow you to add access modifiers to methods :pain:
Everything is private by default, and that's the only option.

Reply:
So extract the processor to a class with a few public methods. That should also make the testing more atomic easier.

Reply:
Will you do this, or should I merge it as-is?

Reply:
I am already doing it in the next PR, I think we can merge this as-is.
See my comment on the next PR